test/OrchardCore.Tests/Data/ContentItemTests.cs

```diff
@@ -155,7 +155,70 @@ public void ShouldDeserializeListContentPart()
 
             var json = JConvert.SerializeObject(contentItem);
 
-            Assert.Contains(@"""MyPart"":{""Text"":""test"",""myField"":{""Value"":123}}", json);
+            var contentItem2 = JConvert.DeserializeObject<ContentItem>(json);
+
+            Assert.NotNull(contentItem2.Content.MyPart);
+            Assert.NotNull(contentItem2.Content.MyPart.myField);
+            Assert.Equal(123, (int)contentItem2.Content.MyPart.myField.Value);
+        }
+
+        [Fact]
+        public void ContentShouldStoreDateTimeFields()
+        {
+            var contentItem = new ContentItem();
+            contentItem.GetOrCreate<MyPart>();
+            contentItem.Alter<MyPart>(x => x.Text = "test");
+            contentItem.Alter<MyPart>(x =>
+            {
+                x.GetOrCreate<MyDateTimeField>("myField");
+                x.Alter<MyDateTimeField>("myField", f => f.Value = new DateTime(2024, 1, 1, 10, 42, 0));
```

Comment:
I'd like to see the same test with a UTC datetime and how it gets back when deserialized.

Reply:
Not really related to this PR, but I added two tests.